src/software/ai/hl/stp/tactic/receiver/receiver_tactic_test.cpp

```diff
@@ -54,11 +54,6 @@ TEST_P(ReceiverTacticTest, perfect_pass_receiver_test)
             // because the robot is free to adjust itself to best receive
             // the pass (and dribble). We only care if the robot received the ball.
             robotReceivedBall(world_ptr, yield);
-
-            while (!tactic->done())
-            {
-                yield("Receiver tactic done but did not receive pass");
-            }
```

Comment:
why did you delete this validation?

Reply:
`ReceiverTactic` runs continuously now and never stops